### PR TITLE
Clarify NG/OK probability normalisation explanation

### PR DESCRIPTION
**Pull request by RyoMurakami1983**
## Summary
- clarify in the probability helper docstring that earlier call sites divided raw logits, causing out-of-range percentages
- explain with an explicit example why the helper now falls back to softmax for negative outputs

## Testing
- python -m py_compile prob_utils.py

------
https://chatgpt.com/codex/tasks/task_b_68dca8d529cc832bba879163b8534997

### CODE REVIEW

**Comment by H-Shibazaki** [comment]
推論メソッドの1本化
